### PR TITLE
A new contributer added

### PR DESCRIPTION
**Pull request by Srijita007**
My first ever PR

### CODE REVIEW

File: src/data/contributors/Srijita.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Srijita Chakrabarty",
+  "skills": ["Python", "C", "HTML/CSS/JAVASCRIPT", "Core Java"],
+  "cause": "Exploring new things",
+  "image": "Avatar2",
+  "github": "",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/Srijita.json:6]: please provide your github url.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
Thanks @Srijita007 for your PR, Welcome to the community!

**Comment by Srijita007** [comment]
i have a doubt. when u said to create different branches for each change, didi that mean changes like the above one also should be done on a new branch? So as to get 2 PRs instead of 2 commits in a single PR?

**Comment by Abhishek-kumar09** [comment]
@Srijita007 
Basically, we should seperate two changes from each other so that they don't  conflicts each other. We can isolate our one code from other by doing things on different branches. That's why branches are introduced.
You can only create one PR from one branch at a time. Once your PR from that branch is merged you could create other PR from the that.
If you want to make two PR at the same time with different code changes you should create two different branches.

Hope it clears your doubts, maybe.

**Pull request merged**